### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/tutorial/static-files.md`

### PR DESCRIPTION
**Pull request by batlopes**
Added translation for pt to docs/pt/docs/tutorial/static-files.md.

### CODE REVIEW

**Review by handreassa — APPROVED** [review]

**Review by cassiobotaro — APPROVED** [review]
lgtm

File: docs/pt/docs/tutorial/static-files.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+```
+
+!!! note "Detalhes técnicos"
+    Você também pode usar `from starlette.staticfiles import StaticFiles`.
+
+    O **FastAPI** fornece o mesmo que `starlette.staticfiles` como `fastapi.staticfiles` apenas como uma conveniência para você, o desenvolvedor. Mas na verdade vem diretamente da Starlette.
+
+### O que é "Montagem"
+
+"Montagem" significa adicionar um aplicativo completamente "independente" em uma rota específica, que então cuida de todas as subrotas.
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/static-files.md:21]: O que acha disso?
```suggestion
"Montagem" significa adicionar um aplicativo completamente "independente" em uma rota específica, que cuida por conta própria de todas as subrotas.
```

File: docs/pt/docs/tutorial/static-files.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Isso é diferente de usar um `APIRouter`, pois um aplicativo montado é completamente independente. A OpenAPI e a documentação do seu aplicativo principal não incluirão nada do aplicativo montado, etc.
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/static-files.md:23]:
```suggestion
Isso é diferente de usar uma `APIRouter`, pois um aplicativo montado é completamente independente. A OpenAPI e a documentação do seu aplicativo principal não incluirão nada do aplicativo montado, etc.
```

File: docs/pt/docs/tutorial/static-files.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+O `directory="static"` refere-se ao nome do diretório que contém seus arquivos estáticos.
+
+O `name="static"` dá a ela um nome que pode ser usado internamente pelo FastAPI.
+
+Todos esses parâmetros podem ser diferentes de "`static`", ajuste-os de acordo com as necessidades e detalhes específicos de sua própria aplicação.
+
+## Mais informações
+
+Para mais detalhes e opções, verifique <a href="https://www.starlette.io/staticfiles/" class="external-link" target="_blank">Starlette's docs about Static Files</a>.
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/static-files.md:39]:
```suggestion
Para mais detalhes e opções, veja na <a href="https://www.starlette.io/staticfiles/" class="external-link" target="_blank">documentação da Starlette sobre arquivos estáticos</a>.
```

**Review by LorhanSohaky — COMMENTED** [review]
Acredito que com essas sugestões a leitura fique mais fluida

**Comment by tiangolo** [comment]
Thank you @batlopes! :cake: 

And thanks for the reviews @LorhanSohaky, @cassiobotaro, and @handreassa :bow: :coffee: 

I'll accept this as it is right now, and maybe another iteration with @LorhanSohaky's suggestions could be made on top, with extra reviews, to keep the conversation about those changes apart but take this as it is for now.

**Pull request merged**